CHANGELOG.md

```diff
@@ -18,7 +18,7 @@
 
 ## New Functionality
 
--
+- [http-agent] Consider `TrustStoreLocation` destination property for self-signed certificate usecase.
```

Comment:
I don't think that people know the "use case". Maybe: ... to support self-signed certificates or something like this?